csharp/Squidex.ClientLibrary/Squidex.ClientLibrary/SquidexAssetClient.cs

```diff
@@ -127,6 +130,24 @@ public async Task<Asset> UpdateAssetContentAsync(string id, string contentName,
             return await response.Content.ReadAsJsonAsync<Asset>();
         }
 
+        public async Task<HttpStatusCode> UpdateAssetAsync(string id, UpdateAssetRequest data)
+        {
+            var jsonData = JsonConvert.SerializeObject(data);
+
+            var request = new StringContent(jsonData, Encoding.UTF8, "application/json");
+            var response = await RequestAsync(HttpMethod.Put, BuildAppAssetsUrl(id), request);
+
+            return response.StatusCode;
+        }
+
+        public async Task<Dictionary<string, string>> GetAssetsTagsAsync()
```

Comment:
Dictionary<string, int> would be better.

Reply:
Fixed.